app/hooks/useScreenSize.tsx

```diff
@@ -0,0 +1,40 @@
+import { useState, useEffect } from "react";
+
+// Define the breakpoints for different screen size categories
+const screenSizes = {
```

Comment:
Use tailwind helper instead of custom sizes with `md: lg:...`.

Reply:
resolved!